WooCommerce/WooCommerceTests/ViewRelated/Products/Actions Factory/ProductFormActionsFactoryTests.swift

```diff
@@ -467,6 +467,18 @@ final class ProductFormActionsFactoryTests: XCTestCase {
         let containsAttributeAction = factory.settingsSectionActions().contains(ProductFormEditAction.attributes(editable: true))
         XCTAssertTrue(containsAttributeAction)
     }
+
+    func test_settings_actions_contain_even_empty_categories_when_it_is_enabled() {
```

Comment:
This looks like it needs to be renamed:

```suggestion
    func test_options_CTA_actions_contain_add_options_when_it_is_enabled() {
```